lib/ansible/modules/remote_management/oneview/oneview_datacenter_facts.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_datacenter_facts
+short_description: Retrieve facts about the OneView Data Centers
+description:
+    - Retrieve facts about the OneView Data Centers.
+version_added: "2.5"
+requirements:
+    - "hpOneView >= 2.0.1"
+author:
+    - Alex Monteiro (@aalexmonteiro)
+    - Madhav Bharadwaj (@madhav-bharadwaj)
+    - Priyanka Sood (@soodpr)
+    - Ricardo Galeno (@ricardogpsf)
+options:
+    name:
+      description:
+        - Data Center name.
+    options:
+      description:
+        - "Retrieve additional facts. Options available: 'visualContent'."
+
+extends_documentation_fragment:
+    - oneview
+    - oneview.factsparams
+'''
+
+EXAMPLES = '''
+- name: Gather facts about all Data Centers
+  oneview_datacenter_facts:
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 500
+  no_log: true
```

Comment:
As mentioned on IRC, please remove this (from here and other examples) and update 
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/oneview.py#L210
to be
`password=dict(type='str', no_log=True),`